SwanCustomEnvironments/swancustomenvironments/serverextension.py

```diff
@@ -0,0 +1,88 @@
+from tornado import web
+from traitlets import Unicode
+from traitlets.config import Configurable
+
+from jupyter_server.base.handlers import JupyterHandler, APIHandler
+from jupyter_server.utils import url_path_join
+
+import subprocess
+import re
+from os import path
+
+
+class SwanCustomEnvironments(Configurable):
+
+    makenv_path = Unicode(
+        path.join(path.dirname(__file__), "scripts/makenv.sh"),
+        config=True,
+        help='Path to the script used to create custom environments.'
+    )
+
+
+class SwanCustomEnvironmentsApiHandler(APIHandler):
```

Comment:
I let it be because we may need more configs in the future. Wyt?

Reply:
And docstring please.

Reply:
@diocas final verdict about this please?

Reply:
I just tried, it seems that class is mandatory to have. I found out that the output wasn't showing up just because the config=None wasn't pre declared as class attribute.

Reply:
Do you understand why it is needed? Did you make it inherit also from Configurable?

Reply:
I've been losing some hours trying to understand what is happening under the hood. For some reason, it fails to create the EventSource calling the api (`serverextension.py`) because it gets a json response from the api (and it is only prepared to get event streamed data)

Reply:
Fixed, I found out the problem was due to the way the variable was being declared. For some reason, I had to give hand of that Unicode function and simply declare it as a normal string

Reply:
So you can have everything in one class now?